### ISSUE TITLE
Metainfo papercuts

### ISSUE THREAD

**Issue by bertob** [andreibachim/shortcut#9]
This app looks good metadata wise, but there are two minor issues:

- The screenshot is too large (recommended window size is 1000x700 px), and doesn't have a window shadow. I'd also recommend having more than 2 items in the list, so it looks less empty.
- No [brand colors](https://docs.flathub.org/blog/introducing-app-brand-colors). Here's a suggestion:

```
<branding>
  <color type="primary" scheme_preference="light">#99c1f1</color>
  <color type="primary" scheme_preference="dark">#1a5fb4</color>
</branding>
```

![image](https://github.com/user-attachments/assets/aba6afd0-3a16-4867-ad5c-a8d9fc4fc03f)

**Comment by andreibachim** [comment]
Thanks for the suggestions, I'll make sure to include them in the next version :saluting_face:

**Comment by andreibachim** [comment]
I applied all the suggestions in version v0.4.2.

Cheers!

**Comment by bertob** [comment]
I had a look at the latest appdata, it's better now but for an app like this with a simple list a narrower screenshot would be better (e.g. 700-800px in width), currently it looks a bit stretched/empty.

**Comment by andreibachim** [comment]
Yeah, you have a point. I will redo them in the next update. Cheers!
